yarn-project/stdlib/src/proofs/proof.ts

```diff
@@ -65,14 +65,15 @@ export class Proof {
     }
     // We are indexing to this particular size because we are assuming the proof buffer looks like:
     // [binary public inputs, binary proof]
```

Comment:
can you add a comment providing some more detail about what happens here in case at some point something breaks